app/web/sentry.js

```diff
@@ -17,8 +17,8 @@ async function createReleaseAndUpload() {
     await cli.releases.new(release);
     console.log("Uploading source maps");
     await cli.releases.uploadSourceMaps(release, {
-      include: ["build/static/js"],
-      urlPrefix: "~/static/js",
+      include: [".next/static/chunks"],
+      urlPrefix: "~/static/chunks",
```

Comment:
```suggestion
      urlPrefix: "~/_next/static/chunks",
```
Based on what I can see from inspecting the dev tool from the preview URL/site

Reply:
Or on second thought, I wonder if we should just leave this script out altogether and let Sentry fetch them automatically as mentioned here for [publicly hosted sourcemaps](https://docs.sentry.io/platforms/javascript/sourcemaps/uploading/hosting-publicly/) 🤔

Reply:
Agreed, I just checked and the source files do have the `# sourceMapping=...` comment